composer.json

```diff
@@ -43,7 +43,7 @@
         "ext-json": "*",
         "ext-mbstring": "*",
         "gettext/gettext": "^4.6",
-        "jaimeperez/twig-configurable-i18n": "^2.0",
+        "jaimeperez/twig-configurable-i18n": "^2.1",
```

Comment:
We have namespaced `jaimeperez/twig-configurable-i18n` to solve some issues regarding inheritance from actual Twig-classes. Psalm doesn't recognize inheritance when mixing namespaced / non-namespaced classes because class-aliases are only created at runtime and Psalm doesn't actually execute code.